modules/identity/src/Volo.Abp.Identity.MongoDB/Volo/Abp/Identity/MongoDB/MongoIdentityUserRepository.cs

```diff
@@ -269,6 +269,25 @@ public MongoIdentityUserRepository(IMongoDbContextProvider<IAbpIdentityMongoDbCo
             .LongCountAsync(GetCancellationToken(cancellationToken));
     }
 
+    public async Task<List<RoleWithUserCount>> GetCountAsync(Guid[] roleIds, CancellationToken cancellationToken = default)
+    {
+        var users = await (await GetMongoQueryableAsync(cancellationToken))
```

Comment:
This won't be efficient.

Reply:
https://github.com/abpframework/abp/pull/17380

Reply:
> This won't be efficient.

https://github.com/abpframework/abp/pull/17380